fastapi/security/api_key.py

```diff
@@ -27,7 +27,7 @@ def __init__(
         self.auto_error = auto_error
 
     async def __call__(self, request: Request) -> Optional[str]:
-        api_key: str = request.query_params.get(self.model.name)
+        api_key = request.query_params.get(self.model.name)
```

Comment:
Why would you remove the typing?

Reply:
The test failed and that's what the previous pr for 0.20.2 did by @Kludex